dashboard/tests/test_helpers.py

```diff
@@ -20,3 +20,9 @@ def test_shorten_node_key_none(self):
             None,
             shorten_node_key(None)
         )
+
+    def test_get_natural_size(self):
+        self.assertEqual(
+            '1.07 GB',
```

Comment:
Discussed with @zolia - it 1GB should be 1024MB, not 1000MB. So this test should equal 1GB.